src/sfProject.ts

```diff
@@ -611,7 +615,15 @@ export class SfProject {
    */
   public getPackage(packageName: string): Optional<NamedPackageDir> {
     const packageDirs = this.getPackageDirectories();
-    return packageDirs.find((packageDir) => packageDir.name === packageName);
+    return packageDirs.find((packageDir) => packageDir.package === packageName || packageDir.name === packageName);
```

Comment:
could this have any unintended consequences? it would be safer to check the original condition first right?

Reply:
I think both are wrong. I am so confused by what to compare in an entry.